pkg/metrics/metrics.jsx

```diff
@@ -257,6 +257,7 @@ class CurrentMetrics extends React.Component {
         this.cgroupMemoryNames = [];
         this.cgroupDiskNames = [];
         this.disksNames = [];
+        this.cpuTemperature = NaN;
```

Comment:
Usually we initialize "empty" values with `null`

Reply:
will do